php/libraries/User.class.inc

```diff
@@ -491,7 +491,8 @@ class User extends UserPermissions
     {
         return in_array(
             $center_id,
-            $this->getCenterIDs()
+            $this->getCenterIDs(),
+            true
```

Comment:
@driusan this is necessary because `in_array(anything, [1,...]) = true` otherwise